### DISCUSSION TITLE
GSoC 2022 Proposals review

### DISCUSSION THREAD

**Discussion by 0904-mansi** [cncf/mentoring#624] [Q&A] [answered]
Hey, @nate-double-u How can we get feedback on our proposals before submitting the proposal? and if CNCF is selected then all projects under the CNCF organization will be also selected? And How many contributors can be selected for one project? I know all questions are stupid but really want to know:(

**Comment by nate-double-u** [comment, marked as answer]
These aren't dumb questions! I think they're actually quite common questions.

> How can we get feedback on our proposals before submitting the proposal?

You shouldn't get feedback before submitting your proposal. Submitting a proposal indicates it's ready for feedback.

Don't wait until the last minute to submit your proposal. The earlier you reach out to the org the more time you will have to incorporate their feedback into your proposal.

> if CNCF is selected then all projects under the CNCF organization will be also selected? 

The CNCF will requests slots, then GSoC will make a decision as to how many to provide. It's possible that there won't be as many slots provided as requested.

The [GSoC Admin tips faq](https://developers.google.com/open-source/gsoc/help/oa-tips#requesting_gsoc_contributor_slots_and_assigning_mentors_to_projects) explains this well.

> And How many contributors can be selected for one project?

[Only an individual may work on a given project](https://developers.google.com/open-source/gsoc/faq#can_a_group_submit_a_proposal_together_to_work_on_a_single_project).

**Reply by 0904-mansi** [reply]
> > if CNCF is selected then all projects under the CNCF organization will be also selected?
> 
> The CNCF will requests slots, then GSoC will make a decision as to how many to provide. It's possible that there won't be as many slots provided as requested.
> 
> The [GSoC Admin tips faq](https://developers.google.com/open-source/gsoc/help/oa-tips#requesting_gsoc_contributor_slots_and_assigning_mentors_to_projects) explains this well.
> >Thanks @nate-double-u . This means that there is no guarantee that the project we have chosen will definitely be selected :(. and if we submitted our proposal then feedback will be provided by mentors so that we can update our proposal accordingly?

**Reply by nate-double-u** [reply]
> This means that there is no guarantee that the project we have chosen will definitely be selected :(. 

That's correct.

> if we submitted our proposal then feedback will be provided by mentors so that we can update our proposal accordingly?

Correct, you can submit updates. Feedback isn't provided on the Summer of Code system though, you'll need to use the same channels you've used to discuss the projects with the mentors to get feedback.
